coreneuron/sim/scopmath/ssimplic_thread.cpp

```diff
@@ -9,11 +9,12 @@
 #include "coreneuron/mechanism/mech/cfile/scoplib.h"
 #include "coreneuron/mechanism/mech/mod2c_core_thread.hpp"
 #include "coreneuron/sim/scopmath/errcodes.h"
+#include "coreneuron/utils/offload.hpp"
 
 namespace coreneuron {
 #define s_(arg) _p[s[arg] * _STRIDE]
 
-#pragma acc routine seq
+nrn_pragma_acc(routine seq)
```

Comment:
`nrn_pragma_omp(declare target)` not needed here?

Reply:
Same as above I think.

Reply:
I added it anyway.